### PR TITLE
[All Hosts] (ui) update React and Fluent article to match new templates

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Also, renames the file (not the article) and moves it to quickstart folder, but it appears in the published ToC in the same place.

**NOTE TO REVIEWERS**:  I did not change the ms.date on files whose only change was to the URL of the updated article.

### CODE REVIEW

File: docs/quickstarts/fluent-react-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+[!INCLUDE [Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
+
+### Create the project
+
+[!INCLUDE [Yeoman generator create project guidance](../includes/yo-office-command-guidance.md)]
+
+- **Choose a project type:** Specify `Office Add-in Task Pane project using React framework`.
+- **Choose a script type:** Specify either `TypeScript` or `JavaScript`.
+- **What do you want to name your add-in?** Specify `My Office Add-in`.
+- **Which Office client application would you like to support?** Specify one of the hosts. (The screenshots in this article will use `Word`. Running the project for the first time is easier if you select `Excel`, `PowerPoint`, or `Word`. See [Try it out](#try-it-out).)
```
Comment (samantharamon) [inline docs/quickstarts/fluent-react-quickstart.md:34]:
```suggestion
- **Which Office client application would you like to support?** Specify one of the hosts. (The screenshots in this article use `Word`. Running the project for the first time is easier if you select `Excel`, `PowerPoint`, or `Word`. See [Try it out](#try-it-out).)
```

File: docs/quickstarts/fluent-react-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+The following is an example.
+
+![The Yeoman generator in a command line interface with the prompts and answers.](../images/yo-office-word-react.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+[!INCLUDE [Yeoman generator next steps](../includes/yo-office-next-steps.md)]
+
+> [!NOTE]
+> Fluent UI React ver. 9 or later isn't supported with the Trident (IE) or EdgeHTML (Edge Legacy) webview controls. If your version of Office is using either of those, the task pane of the add-in generated by Yo Office simply contains a message advising the user to upgrade their version of Office. For more information, see [Browsers and webview controls used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```
Comment (samantharamon) [inline docs/quickstarts/fluent-react-quickstart.md:45]:
```suggestion
> Fluent UI React v9 or later isn't supported with the Trident (IE) or EdgeHTML (Edge Legacy) webview controls. If your version of Office is using either of those, the task pane of the add-in generated by Yo Office simply contains a message to upgrade your version of Office. For more information, see [Browsers and webview controls used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```
For consistency with other version instances in the article.

File: docs/quickstarts/fluent-react-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Explore the project
+
+The add-in project that you've created with the Yeoman generator contains sample code for a basic task pane add-in. If you'd like to explore the components of your add-in project, open the project in your code editor and review the files listed below. The file name extensions depend on which language you choose. TypeScript extensions are in parentheses. When you're ready to try out your add-in, proceed to the next section.
```
Comment (samantharamon) [inline docs/quickstarts/fluent-react-quickstart.md:49]:
```suggestion
The add-in project that you've created with the Yeoman generator contains sample code for a basic task pane add-in. If you'd like to explore the components of your add-in project, open the project in your code editor and review the following files. The file name extensions depend on which language you choose. TypeScript extensions are in parentheses. When you're ready to try out your add-in, proceed to the next section. 
```
To align with MS Style Guide.

File: docs/quickstarts/fluent-react-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+### Explore the project
+
+The add-in project that you've created with the Yeoman generator contains sample code for a basic task pane add-in. If you'd like to explore the components of your add-in project, open the project in your code editor and review the files listed below. The file name extensions depend on which language you choose. TypeScript extensions are in parentheses. When you're ready to try out your add-in, proceed to the next section. 
+
+- The **./manifest.xml** file in the root directory of the project defines the settings and capabilities of the add-in. To learn more about the **manifest.xml** file, see [Office Add-ins XML manifest](../develop/xml-manifest-overview.md).
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane and loads the Office JavaScript Library. It also tests whether the webview control supports Fluent UI React ver. 9 and displays a special message if it doesn't.
```
Comment (samantharamon) [inline docs/quickstarts/fluent-react-quickstart.md:52]:
```suggestion
- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane and loads the Office JavaScript Library. It also tests whether the webview control supports Fluent UI React v9 and displays a special message if it doesn't.
```

File: docs/quickstarts/fluent-react-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+1. A **WebView Stop On Load** prompt appears. Select **OK**.
+
+1. If the "My Office Add-in" task pane isn't already open, choose the **Home** tab, and then choose the **Show Taskpane** button on the ribbon to open the add-in task pane.
+
+1. Enter text into the text box and then select **Insert text**. 
+
+![Custom text inserted into the document after selecting the Insert button from the add-in task pane.](../images/word-task-pane-react-component.png)
+
+### Next steps
```
Comment (samantharamon) [inline docs/quickstarts/fluent-react-quickstart.md:96]: Suggesting to remove this heading, unless we add other recommended steps, such as checking out the Fluent UI React v9 documents.

File: docs/images/word-task-pane-react-component.png
Comment (samantharamon) [inline docs/images/word-task-pane-react-component.png:1]: Should we include the Copilot icon in our screenshots? I believe it's still in preview.

**Review by samantharamon — APPROVED** [review]
Thanks for making these changes, @Rick-Kirkham. Looks good to me. Added minor suggestions.

File: docs/quickstarts/fluent-react-quickstart.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+Congratulations, you've successfully created a task pane add-in using React and Fluent UI React!
```
Comment (Rick-Kirkham) [inline docs/quickstarts/fluent-react-quickstart.md:98]:
```suggestion
```

**Pull request merged**